corgi/collectors/errata_tool.py

```diff
@@ -261,3 +262,15 @@ def _parse_module(
                 # map modular_rpms to build_ids. In slow_save_errata these are only saved as
                 # meta_attr so, let's just save the entire list for each build_id
                 variant_to_component_map[variant].append({build_id: modular_rpms})
+
+    def get_erratum_notes(self, erratum_id: int):
+        # Get the contents of the "Notes" (formerly "How to test") field from an erratum.
+        # Quarkus uses this field to associate an SBOM with an erratum.
+        erratum_json = self.get(f"api/v1/erratum/{erratum_id}?format=json")
+        try:
+            # All products that use SBOMer should have notes set
+            notes = json.loads(erratum_json["content"]["content"]["how_to_test"])
```

Comment:
In the Errata Tool web UI, it's "Notes", but in the json data returned by the API, it's still "how_to_test". I'd talk to the ET team about changing the name of the field in the API, but I'd bet there are other tools out there that would break if they made such a change. Our approach here is (famous last words) temporary, for that reason also I'm not sure it's worth pursuing a change, but I'll add a comment to clarify.